### PR TITLE
chore(configure eslint):

### PR DESCRIPTION
**Pull request by ngireric123**
#### What does this PR do?
Configure ESLint
#### Description of Task to be completed?
* Add .eslintrc.json file
* Install ESLint 
#### How should this be manually tested?
* clone repository 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168781664
#### Screenshots (if appropriate)
Screen Shot 2019-09-30 at 14 15 08

### CODE REVIEW

**Review by Cheza-Dzabala — APPROVED** [review]

**Review by alainmateso — APPROVED** [review]
Looks good!

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          "ignoreRefs": true,
+          "allowArrowFunctions": true,
+          "allowFunctions": true,
+          "allowBind": true,
+          "ignoreDOMComponents": true
+          }],
+          "comma-dangle": "off",
+          "import/prefer-default-export": "off",
+          "no-unused-vars": 0, 
+           "no-console": "off",
```
Comment (nignanthomas) [inline .eslintrc.json]: Does this mean ESLint will allow console.log()?

**Review by nignanthomas — COMMENTED** [review]
We shall not submit PR with console.log()'s
I wanna make sure ***no-console: "off"*** highlights console.log()

**Review by nakiwuge — DISMISSED** [review]
@ngireric123 good job however your task is not fully complete. There are some important rules you have missed out like `console.log()` should give a warning, you have no indentation rule, quotes rule, the `undefined` variable should also give a warning, a line at the end of a file, etc. Kindly look into that.

File: .eslintrc.json (hunk 1 above)
Comment (ngireric123) [reply .eslintrc.json]: @nignanthomas i did a mistake here on no-console let me correct it and some rules

**Pull request merged**